doc/connectivity/bluetooth/api/mesh/shell.rst

```diff
@@ -1170,6 +1170,27 @@ receiving any BLOB data, but the implementation in the mesh shell will discard t
 	Get a list of all BLOB Transfer Server model instances on the node.
 
 
+Binary Large Object (BLOB) Transfer flash stream
+------------------------------------------------
+BLOB flash stream configuration can be added to the mesh shell by enabling the
+:kconfig:option:`CONFIG_BT_MESH_BLOB_IO_FLASH` option. By default, the shell uses a dummy BLOB
```

Comment:
```suggestion
:kconfig:option:`CONFIG_BT_MESH_SHELL_BLOB_IO_FLASH` option. By default, the shell uses a dummy BLOB
```

Reply:
The shell config is set to `y` by default if the other config is set: https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/bluetooth/mesh/shell/Kconfig#L141-L142. For the models the "basic" configs are used in the docs so I was a bit confused on what to choose. Do you still think I should change it?

Reply:
It says: `BLOB flash stream configuration can be added to the mesh **shell** by enabling the`
but shell commands are enabled through `CONFIG_BT_MESH_SHELL_BLOB_IO_FLASH` configuration option:
https://github.com/zephyrproject-rtos/zephyr/blob/f1087d2042236e3e9867baad69f78b24bcd77ce6/subsys/bluetooth/mesh/shell/blob.c#L597C13-L601
Of course, shell commands can't be enabled without the particular feature, but then we could say what is said in other cases here:
`... is an optional Bluetooth Mesh model that can be enabled through the
<kconfig> configuration option`, where `<kconfig>` is the configuration option that enables the feature, not shell commands. For example:
https://github.com/zephyrproject-rtos/zephyr/blob/f1087d2042236e3e9867baad69f78b24bcd77ce6/doc/connectivity/bluetooth/api/mesh/shell.rst?plain=1#L938-L939

I'm not OK with this as well because since we are talking about the shell commands in this doc, we should tell about `..._SHELL_...` configuration options. If I'm not mistaken it is artifacts from previous implementation where there were no `..._SHELL_...` confiugration options and all shell commands were enabled with `CONFIG_BT_MESH_SHELL`.

P.S. Yes, in some places we already incorrect, e.g.: 
https://github.com/zephyrproject-rtos/zephyr/blob/f1087d2042236e3e9867baad69f78b24bcd77ce6/doc/connectivity/bluetooth/api/mesh/shell.rst?plain=1#L1474-L1476
https://github.com/zephyrproject-rtos/zephyr/blob/f1087d2042236e3e9867baad69f78b24bcd77ce6/subsys/bluetooth/mesh/shell/dfu.c#L1006-L1008

Reply:
Fair, added a small commit fixing your example as well